build/package.json

```diff
@@ -85,5 +85,9 @@
     "tree-sitter-typescript": "^0.20.1",
     "vscode-gulp-watch": "^5.0.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@actions/core": "1.2.6",
```

Comment:
@lewis-sanchez These shouldn't be in dependencies - in fact nothing in build should be. The stuff in this folder is only ever used during build time so everything should be under `devDependencies`. Please fix that to avoid conflicts with future VS Code merges.